internal/printer/metadata.go

```diff
@@ -118,3 +131,50 @@ func (m *Metadata) createSummary() (*component.Summary, error) {
 	summary := component.NewSummary("Metadata", sections...)
 	return summary, nil
 }
+
+func (m *Metadata) managedFields() ([]component.Summary, error) {
+	a, err := meta.Accessor(m.object)
+	if err != nil {
+		return nil, err
+	}
+	if len(a.GetManagedFields()) == 0 {
+		return nil, nil
```

Comment:
We are not handling this case in the calling method which will fail to iterate over nil. This is omittable as not having managed fields will just return an empty array.

Reply:
Removed